barcodes_generator_product/tests/test_barcodes_generator_product.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (C) 2016-Today GRAP (http://www.grap.coop)
+# Copyright (C) 2016-Today La Louve (http://www.lalouve.net)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class Tests(TransactionCase):
```

Comment:
```suggestion
class Tests(BaseCommon):
```


Use BaseCommon as base test class to reduce overhead from tracking and boost the test suite.

Reply:
reference: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-18.0#tasks-to-do-in-the-migration